app/src/main/java/com/example/stretchy/features/executetraining/ui/composable/TrainingSummaryComposable.kt

```diff
@@ -0,0 +1,77 @@
+package com.example.stretchy.features.executetraining.ui.composable
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import com.example.stretchy.Screen
+import com.example.stretchy.theme.BananaMania
+
+@Composable
+fun TrainingSummaryComposable(numberOfExercises: Int, timeSpent: String,navController: NavController) {
```

Comment:
`timeSpent` time spent where, time spent of what? be pricse...

`timeSpentExercising` or `currentTrainingTime`